example-apps/chatbot-rag-app/README.md

```diff
@@ -51,14 +51,13 @@ ease while locally is advised if you are making changes to the application.
 ### Run with docker
 
 Docker compose is the easiest way, as you get one-step to:
-* build the [frontend](frontend)
 * ingest data into elasticsearch
 * run the app, which listens on http://localhost:4000
 
 **Double-check you have a `.env` file with all your variables set first!**
 
 ```bash
-docker compose up --build --force-recreate
+docker compose up --force-recreate
```

Comment:
I considered `--pull always` for reproducibility but figured the simpler command is better

Reply:
agreed in principal, but `--force-recreate` is already there, too. I think we should do `--pull-always` until the telemetry stuff settles down, so that a future blog won't result in someone retrying and not seeing the new stuff. I know it may be overly cautious, so not hard opinion on this.

Reply:
Yeah given possible telemetry churn makes sense, added it